### ISSUE TITLE
readme is not synced with makefile

### ISSUE THREAD

**Issue by yuvalif** [todotxt/todo.txt-cli#261]
According to the readme it is possible to control the install directories when running ```make install```, however the makefile does not allow that. The only configurable parameter there is "DESTDIR", but it forces all sub-directories to be under the same one, which is not necessarily the case.

Ideally, the example from the readme is supported in the makefile:
```
make install CONFIG_DIR=/etc INSTALL_DIR=/usr/bin BASH_COMPLETION_DIR=/usr/share/bash-completion/completions
```

**Which versions todo.sh are you using?**
```TODO.TXT Command Line Interface v@DEV_VERSION@```

**Which Operating System are you using?**
Fedora28

**Which version of bash are you using?**
```GNU bash, version 4.4.23(1)-release (x86_64-redhat-linux-gnu)```

**Comment by iamleot** [comment]
Hello Yuval,

Yuval Lifshitz writes:
> According to the readme it is possible to control the install directories when running ```make install```, however the makefile does not allow that. The only configurable parameter there is "DESTDIR", but it forces all sub-directories to be under the same one, which is not necessarily the case.
>
> Ideally, the example from the readme is supported in the makefile:
> ```
> make install CONFIG_DIR=/etc INSTALL_DIR=/usr/bin BASH_COMPLETION_DIR=/usr/share/bash-completion/completions
> ```
>
> **Which versions todo.sh are you using?**
> ```TODO.TXT Command Line Interface v@DEV_VERSION@```
>
>
> **Which Operating System are you using?**
> Fedora28
>
> **Which version of bash are you using?**
> ```GNU bash, version 4.4.23(1)-release (x86_64-redhat-linux-gnu)```
>

JFTR I think that PR #259 addresses that.

**Comment by inkarkat** [comment]
Fixed by #259
